src/operatorApp/tests/operator.test.js

```diff
@@ -180,4 +180,140 @@ describe('Operator router tests', () => {
 			.set('Authorization', `Bearer ${token}`);
 		expect(response).to.have.status(400);
 	});
+	// Testing for logged in as an operator
+	it('should login an operator user', async () => {
+		const password = await bcrypt.hash('Andela@1234', 5);
+		const newUser = User.create({
+			email: 'operator@gmail.com',
+			password,
+			firstName: 'test',
+			lastName: 'Andela',
+			role: 'operator',
+		});
+		const user = await newUser.save();
+		const info = { email: user.email, password: 'Andela@1234' };
+		const response = await chai
+			.request(app)
+			.post('/api/v1/accounts/login')
+			.send(info);
+		expect(response).to.have.status(200);
+		token = response.body.data.access_token;
+	});
+	it('should not create an operator', async () => {
+		operatorInfo = {
```

Comment:
move  in that file this too

Reply:
These are not tests related to this PR actually, they are tests for CRUD operations for bus. I changed these when I edited the expected response from 401 to 403